src/components/PaymentMethodTabPane.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { TabPane } from "reactstrap";
+import { PaymentMethod } from "../generated/definitions/PaymentMethod";
+import { useTranslation } from "react-i18next";
+import { format, parseISO } from "date-fns";
+
+type PaymentMethodTabPaneProps = {
```

Comment:
naming: try to avoid leak implementation details in variable / file naming (what if we refactor this as a Dialog instead of a TabPane ? ), `PaymentMethod` would fit better as component name